cmd/ocm-backplane/testJob/createTestJob.go

```diff
@@ -174,16 +191,28 @@ func createTestScriptFromFiles() (*backplaneApi.CreateTestScriptRunJSONRequestBo
 		logger.Errorf("Error reading metadata: %v", err)
 		return nil, err
 	}
-
+	// /examples/lib-sourcer/script.sh
 	// Now, try to find and read the script body
 	fileBody, err := os.ReadFile(scriptMeta.File)
 
+	// fileBody, err := os.ReadFile(scriptMeta.File)
+	// if file.contains(source /managed-scipts(lib_path)\n)
+	// find the file at repo_root/scripts(lib_path)
+	// add cat << EOF > /managed-scripts/(lib_path) to script before source
```

Comment:
Do we need to include these comments? Can't it be removed instead?

Reply:
We don't need them - good pickup.